server/pulp/server/webservices/views/plugins.py

```diff
@@ -29,4 +30,27 @@ class TypeResourceView(View):
 
 
 class TypesView(View):
-    pass
+    """
+    View for dealing with all plugin types.
+    """
+
+    @auth_required(authorization.READ)
+    def get(self, request, *args, **kwargs):
```

Comment:
I don't think this should have *args and **kwargs. If this view does receive a positional or keyword argument in the future this signature can be adjusted.